app/components/certificate/description.tsx

```diff
@@ -21,14 +21,18 @@ export default function DescriptionSection({
       <Text>{description}</Text>
       {certRequested && (
```

Comment:
I don't think we need `certRequested` if we're going to validate based off `validFrom` and `validTo`. We can move the validation above to start from `<HStack>`